doc/bibliography/published.bib

```diff
@@ -7,6 +7,113 @@
+	Volume = {18},
+	Year = {2020},
+	Bdsk-Url-1 = {https://doi.org/10.5065/g8w0-fy32}}
+
+
+@article{ZannaJansen20,
+	Author = {Zanna, L., S. Bachman and M. Jansen},
+	Date-Added = {2020-03-10 11:24:21 -0400},
+	Date-Modified = {2020-03-10 11:24:21 -0400},
+	Doi = {10.5065/g8w0-fy32},
+	Journal = {CLIVAR Exchanges/US CLIVAR Variations},
+	Month = {February},
+	Number = {1},
+	Pages = {3--8},
+	Title = {Energizing turbulence closures in ocean models},
+	Url = {https://doi.org/10.5065/g8w0-fy32},
+	Volume = {18},
+	Year = {2020},
+	Bdsk-Url-1 = {https://doi.org/10.5065/g8w0-fy32}}
+
+
+@article{Grooms20,
+	Author = {Grooms, I.},
+	Date-Added = {2020-03-10 11:24:51 -0400},
+	Date-Modified = {2020-03-10 11:24:51 -0400},
+	Doi = {10.5065/g8w0-fy32},
+	Journal = {CLIVAR Exchanges/US CLIVAR Variations},
+	Month = {February},
+	Number = {1},
+	Pages = {9--13},
+	Title = {Mesoscale eddy energy transport},
+	Url = {https://doi.org/10.5065/g8w0-fy32},
+	Volume = {18},
+	Year = {2020},
+	Bdsk-Url-1 = {https://doi.org/10.5065/g8w0-fy32}}
+
+
+@article{ColeAbernathey20,
+	Author = {Cole, S. T. and K. Drushka and R. Abernathey},
+	Date-Added = {2020-03-10 11:22:50 -0400},
+	Date-Modified = {2020-03-10 11:26:24 -0400},
+	Doi = {10.5065/g8w0-fy32},
+	Journal = {CLIVAR Exchanges/US CLIVAR Variations},
+	Month = {February},
+	Number = {1},
+	Pages = {37--41},
+	Title = {GEOMETRIC: Geometry and energetics of ocean mesoscale eddies and their representation in climate models},
+	Url = {https://doi.org/10.5065/g8w0-fy32},
+	Volume = {18},
+	Year = {2020},
+	Bdsk-Url-1 = {https://doi.org/10.5065/g8w0-fy32}}
+
+
+@article{ChassignetYeager20a,
+	Author = {Chassignet, E. P. and S. G. Yeager and B. Fox-Kemper and A. Bozec and F. S. Castruccio and G. Danabasoglu and W. M. Kim and N. Koldunov and Y. Li and P. Lin and H. Liu and D. Sein and D. Sidorenko and Q. Wang and X. Xu},
+	Doi = {10.5065/g8w0-fy32},
+	Journal = {CLIVAR Exchanges/US CLIVAR Variations},
+	Title = {Impact of Horizontal Resolution on the energetics of global ocean-sea-ice model simulations},
+	Url = {https://doi.org/10.5065/g8w0-fy32},
+	Volume = {18},
+	Year = {2020},
+	Bdsk-Url-1 = {https://doi.org/10.5065/g8w0-fy32}}
+
+
+@article{ColeAbernathey20,
+	Author = {Cole, S. T. and K. Drushka and R. Abernathey},
+	Date-Added = {2020-03-10 11:22:50 -0400},
+	Date-Modified = {2020-03-10 11:26:24 -0400},
+	Doi = {10.5065/g8w0-fy32},
+	Journal = {CLIVAR Exchanges/US CLIVAR Variations},
+	Month = {February},
+	Number = {1},
+	Pages = {37--41},
+	Title = {GEOMETRIC: Geometry and energetics of ocean mesoscale eddies and their representation in climate models},
+	Url = {https://doi.org/10.5065/g8w0-fy32},
+	Volume = {18},
+	Year = {2020},
+	Bdsk-Url-1 = {https://doi.org/10.5065/g8w0-fy32}}
+
+
 @article{TarshishEtAl2018,
 	Author = {Tarshish, Nathaniel and Abernathey, Ryan and Zhang, Ci and Dufour, Carolina O and Frenger, Ivy and Griffies, Stephen M},
 	Date-Added = {2018-09-22 14:16:54 +0000},
```

Comment:
Now that we have some real publications in here, we might as well remove these dummy publications I put in as placeholders. Would you mind doing that as part of your PR?